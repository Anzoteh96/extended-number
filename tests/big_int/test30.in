-3688706187166874468278500826052394702400187112003880920154239449693992383208526794870468165
-461901550432760274692266670684113761448051273848426823545754524908179177736747901788615718
