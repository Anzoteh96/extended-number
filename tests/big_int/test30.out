Enter two numbers n and m:
n is -3688706187166874468278500826052394702400187112003880920154239449693992383208526794870468165
m is -461901550432760274692266670684113761448051273848426823545754524908179177736747901788615718
n + m is -4150607737599634742970767496736508463848238385852307743699993974602171560945274696659083883
n - m is -3226804636734114193586234155368280940952135838155454096608484924785813205471778893081852447
n * m is 1703819106943294928370665593167593407084704149656582199300209881582222680107930050383732615320959146571319234542608791907328341203037566303336468997860504687998694510084633937617470
n / m is 7
n % m is -455395334137552545432634131263598372263828195064893155333957775336738139051291482350158139
